core/kazoo_modb-1.0.0/priv/couchdb/views/notify.json

```diff
@@ -0,0 +1,9 @@
+{
+   "_id": "_design/notify",
```

Comment:
Can we call it the `_design/notifications` view so that folks don't think it is tied to the 'notify' app?